.gitlab-ci.yml

```diff
@@ -165,6 +193,7 @@ build:image:ubuntu:focal:arm64:
     - docker load -i builder-$DISTRO-$RELEASE-$ARCH.tar
   script:
     - apk --update --no-cache add bash
+    - find output
```

Comment:
Minor: Isn't this more useful after the building steps?

Reply:
moved down. it makes sense in both places, before: so we know that the required files are there and after so we know what we have built.